backend/src/routes/stats.py

```diff
@@ -0,0 +1,58 @@
+from time import time
+import json
+
+from fastapi import APIRouter, HTTPException
+from starlette.status import HTTP_400_BAD_REQUEST
+from starlette.responses import JSONResponse
+from starlette.requests import Request
+from mongoengine import DoesNotExist
+
+from ..models import UserStats
+from ..auth import (
+    verify_cookies,
+    CookieVerificationError,
+)
+from ..game import Player
+
+stats_router = APIRouter()
+
+
+@stats_router.get("/stats")
+async def get_stats(request: Request) -> JSONResponse:
+    try:
+        username = verify_cookies(request.cookies, "access_token")
+        user = UserStats.objects.get(username=username)
+        player_stats = json.loads(user.to_json())
+        response_json = {}
+        del player_stats["_id"]
+        response_json["top5"] = get_current_top()
+        response_json["player_stats"] = player_stats
+        response = JSONResponse(response_json)
+        return response
+    except (CookieVerificationError, DoesNotExist):
+        raise HTTPException(status_code=HTTP_400_BAD_REQUEST)
+
+
+def get_current_top():
+    players = UserStats.objects.order_by("-games_won").limit(5)
+    players.to_json()
```

Comment:
Does this line do anything? You are not assigning it.
Actually I think You could do sth like this:
```python
json.loads(players.to_json)
```
If your goal is to return a dictionary, then this is a way to go.

Reply:
You're right - I've deleted this line, but i can't just return `players.to_json` - i have to delete specific fields (like _id, heals) before sending it.